recipes/seadex-essentials/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import export_conandata_patches, get, copy, rename
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class SeadexEssentialsConan(ConanFile):
+    name = "seadex-essentials"
+    description = "essentials is a small c++ library that offers very basic capabilities for applications and libraries."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://essentials.seadex.de/"
+    topics = ("utility", "c++")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "spdlog/*:header_only": True
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "8.3",
+            "clang": "12",
+            "Visual Studio": "16",
+            "msvc": "192",
+            "apple-clang": "10"
+        }        
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            self.options.shared = False
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.options["spdlog/*"].fmt_external = True
```

Comment:
I do not see this option https://github.com/prince-chrismc/conan-center-index/blob/9b230a576b732ca85edaf9f2a2389a169a3b5275/recipes/spdlog/all/conanfile.py#L22 🤔